README.md

```diff
@@ -331,3 +420,33 @@
   ```javascript
   () => {}
   ```
+
+#### Now
+
+* Underscore
+
+  ```javascript
+  _.now()
+  ```
+
+* ES5.1
+
+  ```javascript
+  Date.now()
+  ```
+
+#### Template
+
+* Underscore
+
+  ```javascript
+  _.template("hello: <%= name %>");
+  compiled({name: 'moe'});
```

Comment:
The variable `compiled` is currently not defined. I think this should be:
```javascript
var greeting = _.template("hello <%= name %>");
greeting({ name: 'moe' });
```

To make the examples match more closely, maybe we could create a template function in the ES2015 example as well, e.g.
```javascript
const greeting = ({ name }) => `hello ${name}`;
greeting({ name: 'moe' });
```